gems/aws-sdk-core/lib/aws-sdk-core/plugins/retry_errors.rb

```diff
@@ -167,8 +167,8 @@ def self.resolve_retry_mode(cfg)
         # Raise if provided value is not one of the retry modes
         if value != 'legacy' && value != 'standard' && value != 'adaptive'
           raise ArgumentError,
-                'Must provide either `legacy`, `standard`, or `adaptive` for '\
-                'retry_mode profile option or for ENV[\'AWS_RETRY_MODE\']'
+            'Must provide either `legacy`, `standard`, or `adaptive` for '\
```

Comment:
`Rubocop` would yell at this change doesn't it?

Reply:
I haven't enabled `Rubocop` for myself yet.  
I align strings based on the review comment alone.  
Do you want to change something?